README.md

```diff
@@ -193,6 +196,42 @@ EOF
 
 The above CR can be queried by `kubectl get otelinst`.
 
+#### NodeJS
+
+The operator can inject OpenTelemetry [NodeJS auto-instrumentation](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/metapackages/auto-instrumentations-node).
+The injection of the NodeJS instrumentation can be enabled by adding an annotation to the namespace, so that all pods within that namespace will get the instrumentation, or by adding the annotation to individual PodSpec objects, available as part of Deployment, Statefulset, and other resources.
+
+```bash
+instrumentation.opentelemetry.io/inject-nodejs: "true"
+```
+
+The value can be
+* `"false"` - do not inject
+* `"true"` - inject and `Instrumentation` resource from the namespace.
+* `"nodejs-instrumentation"` - name of `Instrumentation` CR instance.
+
+In addition to the annotation, the following `CR` has to be created. The `Instrumentation` resource provides configuration for OpenTelemetry SDK and auto-instrumentation.
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
```

Comment:
I think we should have only a single instrumentation CR in the readme and then have a bulletoints/section for each language with annotations.

Reply:
Tweaked the README - hoping to be able to follow up with additional doc nits since it's been a big struggle keeping up with merge conflicts.

Reply:
Agree I had to rebase yesterday like 100k times